community/portworx/templates/portworx-ds.yaml

```diff
@@ -179,17 +164,17 @@ spec:
                   key: ".dockerconfigjson"
                   name: "{{ $registrySecret }}"
             {{- end }}
-           {{- if not (eq $etcdCredentials "none") }}
+           {{- if not (eq $etcdSecret "none") }}
             - name: ETCD_USERNAME
               valueFrom:
                 secretKeyRef:
                   key: "username"
-                  name: "{{ $etcdCredentials }}"
+                  name: "{{ $etcdSecret }}"
```

Comment:
Let's confirm with the IBM team, that forcing TLS is expected or not, otherwise we should support passing the credentials in as `username:password`.
We should have both the options open as this chart can be used outside the OSB too, and user's might want to setup basic authentication.

Reply:
The etcd from IBM only supports TLS and setting that up is documented: https://cloud.ibm.com/docs/containers?topic=containers-portworx#portworx_database

This chart is only for IBM cloud, we should control the environment and ask users to use the etcd hosted by IBM services to reduce varying configs.

Reply:
Ok, so if the user wants to install using helm they can still use an external ETCD if they want and via OSB we force ETCD to be encrypted.